CONTRIBUTING.md

```diff
@@ -17,23 +17,38 @@ If you would like to contribute to this repository by adding features, enhanceme
 
 ## Table of Contents
 
-- [Opening an Issue](#opening-an-issue)
-  - [Grammar](#grammar)
+- [Development Setup](#development-setup)
 - [Pull Request Guidelines](#pull-request-guidelines)
   - [Documentation](#documentation)
   - [Testing](#testing)
   - [Code Style](#code-style)
   - [Git Commit](#git-commit)
 - [Release Process](#release-process)
 
-## Opening an Issue
-The title of a Bug or Enhancement should clearly indicate what is broken or desired. Use the description to explain possible solutions or add details and (especially for Enhancements) explain *how* or *why* the issue is broken or desired. Follow the template!
+## Development Setup
+### Getting Started
+To get started developing for contributions, follow these steps:
 
-### Grammar
-While quibbling about grammar in issue titles may seem a bit pedantic, adhering to some simple rules can make it much easier to understand a Bug or an Enhancement from the title alone. For example, is the title **"Browsers should support blinking text"** a bug or a feature request?
+1. Fork the component repository
 
-- Enhancements: The title should be an imperative statement of how things should be. **"Add support for blinking text"**
-- Bugs: The title should be a declarative statement of how things are. **"Text does not blink"**
+    1. Click on the `Fork` button on the top-right corner
+    2. Wait for Github to finish forking the repository
+    3. Head over to your newly created repository fork
+
+2. Clone your repository fork locally
+
+    ```bash
+    git clone git@github.com:{your username}/components.git
+    ```
+
+3. Install all project dependencies
```

Comment:
Does `npm-install-peers` do a full install as well?

Reply:
Yes, install peers + others

Reply:
Could we change this to 
```suggestion
3. Install all project dependencies and peers
```
to avoid confusion?

Reply:
Updated